duqtools/create.py

```diff
@@ -0,0 +1,71 @@
+import itertools
+from enum import Enum
+from logging import debug
+from typing import List
+
+from pydantic import BaseModel, DirectoryPath
+
+import duqtools.config
+
+from .ids import write_ids
+from .jetto import JettoSettings
+
+
+class Sources(Enum):
```

Comment:
Does it have to be an Enum? Good for now, but something to think about when/if we want to support other systems

Reply:
Yeah, the other suggested alternative was `typing.Literal`, but that's not available in Python 3.7. I'd be happy if there was something else we could use.